gwtp-core/gwtp-dispatch-rest/src/main/java/com/gwtplatform/dispatch/rest/client/core/DefaultResponseDeserializer.java

```diff
@@ -103,11 +101,16 @@ private boolean isSuccessStatusCode(Response response) {
             Serialization serialization = findSerialization(resultClass, contentType);
 
             if (serialization != null) {
-                return deserializeValue(serialization, resultClass, contentType, response.getText());
+                try {
```

Comment:
Actaully this one needs the response data so it has to go outside it or the response needs to be passed in.